File: simulation/SensorData.js
"use strict";

class SensorData{
	
	constructor(rotation_matrix, rotation_matrix_line, gyro, height, d_height){
		this.rotation_matrix = rotation_matrix;
		this.rotation_matrix_line = rotation_matrix_line;
		this.gyro = gyro;
		this.height = height;
		this.d_height = d_height;
	}
	
}

File: mpu-testing/main/control/sensorData.c
#include "sensorData.h"
#include <string.h>

void initSensorData(SensorData* sensorData, float* rotation_matrix, float* rotation_matrix_line, float* gyro, float height, float d_height){
	sensorData->rotation_matrix = rotation_matrix;
	sensorData->rotation_matrix_line = rotation_matrix_line;
	sensorData->gyro = gyro;
	sensorData->height = height;
	sensorData->d_height = d_height;
}
